osutils.pm

```diff
@@ -74,4 +75,18 @@ sub qv($) {
 }
 ## use critic
 
+# runcmd use SIGCHLD handler to wait all child processes at exit and reap zombies
+sub runcmd {
+    bmwqemu::diag "running " . join(' ', @_);
+    my %children;
+    local $SIG{CHLD} = sub {
+        local ($!, $?);
+        while ((my $pid = waitpid(-1, WNOHANG)) > 0) {
+            delete $children{$pid};
+            cleanup_child($pid, $?);
+        }
+    };
+    return CORE::system(@_);
```

Comment:
Wouldn't it be safer to throw an exception if the call fails so we don't need to check the exit code explicitly on every call?